src/modules/eventListeners.js

```diff
@@ -0,0 +1,57 @@
+import {
+  renderToDoList,
+  addTask,
+  clearCompleted,
+  editTask,
+  deleteTask,
+  markTask,
+} from './listUtils.js';
+import { updateLocalStorage } from './localStorage.js';
+import toDoTasks from '../toDoTasks.js';
+
+const input = document.querySelector('.task-input');
+const todoList = document.querySelector('.todo-list-ul');
+const clearCompletedBtn = document.querySelector('.todo-list-clear-btn');
+const addTaskBtn = document.querySelector('.task-add-btn');
+
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
```

Comment:
Kindly note that, as part of JavaScript best practices, negative conditionals must be avoided. Positive conditionals a make the code more explicit and readable.

Reply:
Let's meet on zoom so we learn about that together. But in the meant time you rewrite the above as;
if (e.key === 'Enter') {
    if (input.value) {
      addTask(toDoTasks, input.value);
      input.value = '';